laboratory-service/src/main/java/com/raf/si/laboratoryservice/service/impl/LabExamServiceImpl.java

```diff
@@ -41,12 +41,6 @@ public LabExamServiceImpl(ScheduledLabExamRepository scheduledLabExamRepository,
     public LabExamResponse createExamination(CreateLabExamRequest createLabExamRequest) {
         checkExamDates(createLabExamRequest.getScheduledDate());
 
-        Optional<Referral> referral = referralRepository.findByLbp(createLabExamRequest.getLbp());
-        if (referral.isEmpty()) {
-            log.error("Ne postoji uput za pacijenta sa ličnim brojem '{}'", createLabExamRequest.getLbp());
-            throw new NotFoundException("Uput sa datim lbp-om ne postoji");
-        }
-
```

Comment:
ova provera nije imala smisla jer pacijent moze da ima vise uputa i onda pukne "query did not return a unique result" kad se pokusa napraviti lab. exam za pacijenta sa vise od jednog uputa